src/os/bluestore/BlueFS.cc

```diff
@@ -3788,7 +3788,9 @@ uint8_t OriginalVolumeSelector::select_prefer_bdev(void* hint)
 void OriginalVolumeSelector::get_paths(const std::string& base, paths& res) const
 {
   res.emplace_back(base, db_total);
-  res.emplace_back(base + ".slow", slow_total);
+  res.emplace_back(base + ".slow",
+    slow_total ? slow_total : db_total); // use fake non-zero value if needed to
```

Comment:
I think supplying "1" is better then db_total.
Works, and at least does not signal RocksDB that there is space on db.slow.